forgerock-authenticator/src/main/java/org/forgerock/android/auth/MechanismParser.java

```diff
@@ -93,6 +93,9 @@ private Map<String, String> map(URI uri) throws MechanismParsingException {
         for (String query : queryParts) {
             String[] split = split(query, "=");
             if (split != null) {
+                if (split[0].equals(ISSUER) && split[1].isEmpty()) {
```

Comment:
can we check this as well  - if (split != null && split.length > 1) . just for safer side to avoid index out of bounds